added for CR late bill part and improved docs for cost management (#934)
meshcloud/meshcloud-docs: docs/meshcloud.payment-methods.md

1. A payment method is **always** scoped to a single meshWorkspace. This means it is not possible to create a global payment method that can be used for all meshWorkspaces. It is also not possible to re-use a single payment method across multiple meshWorkspaces.
2. A payment method's identifier is globally unique. This means you cannot create a payment method with the same identifier for multiple meshWorkspaces. Make sure to use an identifier that will not collide in other meshWorkspaces.

> >Please note that payment methods by now are supporting only EUR amounts.

Reviewer: ```suggestion
> Please note that as of today payment methods only support EUR amounts. 
```

### Creating a Payment Method via the meshPanel

One way of creating payment methods is via the meshPanel. To do so, make sure that you have an account with the 'Partner Admin' role. Navigate to the Admin area and follow these steps:

> 1. Go to 'Payment Method' page under the FinOps section.

Reviewer: You're missing a step after 1. which is selecting the workspace for which the payment method will be created.

Select the Active Payment Method of choice, and optionally a Substitute Payment Method if that is desired. Save the meshProject and the payment method(s) will be assigned to the meshProject.

> > Please not expired payment methods can not be assigned to projects anymore and are removed from project when expired.

Reviewer: ```suggestion
> Please note that expired payment methods are removed from all projects automatically upon expiration and can no longer be assigned to any projects.
```
